module/VuFind/src/VuFind/Db/Entity/ExchangeArrayInterface.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * Interface for representing a user account record.
```

Comment:
Maybe this would be more accurate:

```suggestion
 * Interface for translating an entity to and from array format.
```